### PR TITLE
Shade deck resize

### PR DESCRIPTION
**Pull request by daschuer**
This makes the Shade deck resizable. 

A attempt to fix bug: 
https://bugs.launchpad.net/mixxx/+bug/1710724

### CODE REVIEW

**Comment by ronso0** [comment]
Cool. Now it looks a bit funny if you enable effect units, but right now I can't suggest a solution other than just stretching the units while keeping the effects centered.

**Comment by ronso0** [comment]
A different issue: FX section in mixer doesn't show tooltips.

**Comment by daschuer** [comment]
OK, I think I should stretch FX and sampler section as well. 

> A different issue: FX section in mixer doesn't show tooltips.

This is fixed in https://github.com/mixxxdj/mixxx/pull/1332 also waiting for a review.

**Comment by daschuer** [comment]
Ready: Fx-Unit and Sampler are skaling as well now.

File: res/skins/Shade/effectunit.xml [hunk 1]
```diff
@@ -66,6 +69,23 @@
 						</Connection>
 					</WidgetGroup>	
 				</Children>		
+			</WidgetGroup>
+			<WidgetGroup>
+		        <ObjectName>EffectRightBorder</ObjectName>
+				<Size>0e,100f</Size>
+				<BackPath>style/style_bg_effectunit_right.png</BackPath>
```
Comment (ronso0) [inline res/skins/Shade/effectunit.xml]: Bg image is not available.
git add -A ?
shade-resize

**Comment by ronso0** [comment]
Tooltips:
"EffectSlot_enabled" [here](https://github.com/mixxxdj/mixxx/blob/master/res/skins/Shade/effect_selector_button.xml#L26)
[here](https://github.com/mixxxdj/mixxx/pull/1333/files#diff-22249b1196e08e4f77946a4024e843acR107), "beatloop" for fixed-size loop buttons is deprecated I guess, maybe "beatloop_activate" is okay even though there's no beat spinbox?

**Comment by Be-ing** [comment]
The changes LGTM.

**Comment by daschuer** [comment]
Thank you for testing!

**Pull request merged**